packages/ui-core/package.json

```diff
@@ -35,6 +35,7 @@
     "styled-components": ">=5"
   },
   "dependencies": {
+    "@keen.io/charts": "^2.31.2",
```

Comment:
This will create circular dependency as have already dependency on `@keen.io/ui-core`